astropy/io/ascii/ui.py

```diff
@@ -144,13 +144,13 @@ def get_reader(Reader=None, Inputter=None, Outputter=None, **kwargs):
     names : list
         List of names corresponding to each data column
     include_names : list
-        List of names to include in output (default= ``None`` selects all names)
```

Comment:
This is a weird format (``default=``), but if we keep it I think it should follow exactly with the format it requires without any ticks or links, etc. Having the space between the ``=`` and the value is, I think, both ugly and confusing.